roles/dev/tasks/db.yml

```diff
@@ -0,0 +1,79 @@
+# Add test databases for middleware to allow functional testing in development environment
```

Comment:
You could add an explanation why the ACL settings are loosened for test purposes. This might be valuable information at a later stage.

Reply:
Thanks for updating the documentation in the functional-tesing.md.
I've added extra documentation to the db.yml
I've deliberately added the drop schema sql sstatement to prevent coupling between the tests and the used version of a specific component.